exporter/fileexporter/factory.go

```diff
@@ -163,10 +168,16 @@ func newFileExporter(conf *Config) (FileExporter, error) {
 	}, nil
 }
 
-func newFileWriter(path string, rotation *Rotation, flushInterval time.Duration, export exportFunc) (*fileWriter, error) {
+func newFileWriter(path string, fileWriteMode string, rotation *Rotation, flushInterval time.Duration, export exportFunc) (*fileWriter, error) {
 	var wc io.WriteCloser
 	if rotation == nil {
-		f, err := os.OpenFile(path, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0644)
+		fileFlags := os.O_RDWR | os.O_CREATE
```

Comment:
Looks like the append mode only works if rotation is not enabled? If so, it should be documented. Configuration the enables rotation and append mode should probably be invalid?

Reply:
Added this to the docs and added a check for it as well.